lib/wx/c_src/egl_impl.h

```diff
@@ -112,9 +112,15 @@ typedef long int int32_t;
 typedef long long int int64_t;
 typedef unsigned long long int uint64_t;
 #elif defined(WIN32) && defined(_MSC_VER)
+#if _MSC_VER == 1900 || _MSC_VER == 1910
+typedef int int32_t;
+typedef __int64 int64_t;
+typedef unsigned __int64 uint64_t;
+#elif _MSC_VER < 1900
 typedef long int int32_t;
```

Comment:
I was only copying the new static types from the new vs 2017 header file. My knowledge doesn't quite up to par with the your change :-(